TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/IntakeRunningCode.java

```diff
@@ -19,7 +21,10 @@ public void init() {
 
         /** DcMotorEx is a child class of DcMotor, so for now we just introduce them as DcMotors.*/
         robot.introduce(new HardwareElement<>(DcMotor.class, hardwareMap, "intakeMotor"));
-
+        /**Introduced Servos*/
+        robot.introduce(new HardwareElement<>(Servo.class, hardwareMap, "geekoWheelServo"));
+        robot.introduce(new HardwareElement<>(Servo.class, hardwareMap, "entraptionServo1"));
```

Comment:
Are there two separate servos controlling two entraption stars of the same purpose? Do they always run at the same time?